docs/userguide/declarative_config.rst

```diff
@@ -24,8 +24,8 @@ boilerplate code in some cases.
     ``setup.cfg`` sections used by the ``pbr`` and ``d2to1`` packages.
 
     Namely: only metadata-related keys from ``metadata`` section are supported
-    (except for ``description-file``); keys from ``files``, ``entry_points``
-    and ``backwards_compat`` are not supported.
```

Comment:
I think this comment is pertinent to the "compatibility with distutils2-like" sections. Is the `entry_points` as implemented by setuptools declarative config compatible with pbr and d2to1?

Reply:
Perhaps better would be to just remove the considerations about pbr and d2to1 compatibility.

Reply:
Sorry I saw your review the other day, but I have been pretty busy, and forgot to reply.

I do think the compatibility part is a bit confusing, especially because it is at the forefront of the page. I have been periodically coming to this page for years now, generally for a quick reminder of some syntax when in doubt, or to see if something previously unsupported got implemented. But only now, and because of your comment, do I realize this particular section was really about compatibility with other tools.

I don't know if it's just me, but when I come to setuptools' documentation, what I expect to find, before anything else, is documentation about setuptools, not other tools. So I am not sure that should be "front matter". Also, since the early days of the setuptools' support for declarative configuration, quite a bit of time has passed. Such matters were probably more of an issue then, than now (though I could be wrong on about that).

So maybe it is now time to revamp this section a bit ? I would keep the compatibility section about those other tools though, because it might still be relevant to some. But I would move it to the bottom of the page. Focus on setuptools' features first. A more comprehensive, independent, "migration" or "cross tool compatibility" section could prove useful, but I wouldn't feel competent enough to write it. I, for example, never used neither `pbr` nor `d2to1`.

I'd be glad to spend some time on this tomorrow, if you find that what I am suggesting makes any sense at all.

Reply:
Your instincts align with mine. Yes, moving that compatibility information into a secondary section makes sense. I too am not particularly familiar with those concerns, so don't have much to add. Thanks for the consideration.